src/ShopUuidManager.php

```diff
@@ -106,4 +106,60 @@ public function isSetForShop($idShop)
 
         return (bool) $this->getForShop($idShop);
     }
+
+    /**
+     * Get one Shop Url
```

Comment:
Goal of this class is only to manage ShopUuid, I thinks we need a ShopProvider or something like that to retrieve url, protocol etc

Reply:
Moved to ShopProvider class ;)